test/blocs/drive_attach_cubit_test.dart

```diff
@@ -68,7 +122,109 @@ void main() {
     );
 
     blocTest<DriveAttachCubit, DriveAttachState>(
-      'set form "${AppValidationMessage.driveNotFound}" error when no valid drive could be found',
+      'driveKey field is added to form when private drive id is used',
+      build: () => driveAttachCubit,
+      setUp: () {},
+      act: (bloc) {
+        bloc.form.value = {
+          'driveId': validPrivateDriveId,
+        };
+        bloc.form.updateValueAndValidity();
+      },
+      wait: Duration(milliseconds: 1000),
+      verify: (bloc) {
+        expect(bloc.form.contains('driveKey'), isTrue);
+      },
+    );
+    blocTest<DriveAttachCubit, DriveAttachState>(
```

Comment:
Nit: missing newline
```suggestion
    );
    
    blocTest<DriveAttachCubit, DriveAttachState>(
```

Reply:
[Added](https://github.com/ardriveapp/ardrive-web/pull/376/commits/b93f655850a4c3114512f04e2019d2b6721727ad)